cli/src/cmd/pkg/down.go

```diff
@@ -0,0 +1,62 @@
+package pkg
+
+import (
+	"cli/core"
+	"context"
+
+	"github.com/luno/jettison/log"
+	"github.com/spf13/cobra"
+)
+
+func PackageDownCommand() *cobra.Command {
```

Comment:
I think you make a fair point, I've applied it in [0d8463e](https://github.com/openhie/package-starter-kit/pull/19/commits/0d8463e90b5fd685634d5a12b705f304fb18350e), just note that I had to refactor it the way I did specifically because of the PackageRemoveCommand()

Reply:
cool nice 👯‍♀️